lib/controllers/esri-token-auth.js

```diff
@@ -1,7 +1,10 @@
 /* jshint node: true, esnext: true */
 "use strict";
+let got;
+(async () => {
+    got = await import("got");
```

Comment:
It's almost been a year since I did this so I don't remember why I did a dynamic import, but it's possible these could be replaced with regular imports if #173 was merged first.